data-prepper-plugins/cloudwatch-logs/src/test/java/org/opensearch/dataprepper/plugins/sink/threshold/ThresholdCheckTest.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.sink.threshold;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.opensearch.dataprepper.plugins.sink.config.ThresholdConfig;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+
+public class ThresholdCheckTest {
+    private ThresholdCheck thresholdCheck;
+
+    @BeforeEach
+    void setUp() {
+        thresholdCheck = new ThresholdCheck(ThresholdConfig.DEFAULT_BATCH_SIZE, ThresholdConfig.DEFAULT_EVENT_SIZE,
+                ThresholdConfig.DEFAULT_SIZE_OF_REQUEST, ThresholdConfig.DEFAULT_LOG_SEND_INTERVAL_TIME);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {55, 80, 100})
+    void check_max_event_size_invalid(final int event_size) {
+        assertThat(thresholdCheck.checkGreaterThanMaxEventSize(event_size), is(true));
```

Comment:
Can be changed to assertTrue.
Lets also try to move `checkGreaterThanMaxEventSize()` out of the assert statement for better readability.

Reply:
I have made changes to make the code more readable.